wallet/src/de/schildbach/wallet/ui/dashpay/DashPayViewModel.kt

```diff
@@ -69,6 +71,19 @@ open class DashPayViewModel @Inject constructor(
 
     val recentlyModifiedContactsLiveData = MutableLiveData<HashSet<String>>()
 
+    private var timerUsernameSearch: AnalyticsTimer? = null
+
+    fun startUsernameSearchTimer() {
+        timerUsernameSearch = AnalyticsTimer(analytics, log, AnalyticsConstants.Process.PROCESS_USERNAME_SEARCH_UI)
+    }
+
+    fun reportUsernameSearchTime(resultSize: Int, searchTextSize: Int) {
+        timerUsernameSearch?.logTiming(
+            Pair("resultCount", resultSize),
+            Pair("searchCount", searchTextSize)
+        )
```

Comment:
I wonder if it's not better to use a map, like this:
```
timer.logTiming(
    mapOf(
        "resultCount" to resultSize,
        "searchCount" to searchTextSize
    )
)
```
Then in the `logTiming` you can have `forEach { (key, value) -> }` which is more verbose than `it.first, it.second`

Reply:
Thanks I will make this change because a map seems easier to work with than `vararg`s.